src/braket/circuits/observables.py

```diff
@@ -77,6 +77,9 @@ def basis_rotation_gates(self) -> Tuple[Gate]:
     def eigenvalues(self) -> np.ndarray:
         return np.array([1, 1])
 
+    def eigenvalue(self, index: int) -> float:
+        return 1
```

Comment:
```suggestion
        return 1.
```

Just to ensure it returns a float and not an int.